wiremock/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:17-jdk-alpine
```

Comment:
Out of curiosity, do we need the full jdk here? We leverage `17-jre-alpine` in the prod dockerfile, and it may save us some space here.

Reply:
I went ahead and removed this file per @alismx suggestion to use the official wiremock image.